webpack.config.ts

```diff
@@ -54,6 +54,9 @@ function resolve(loggerName: string): ResolveOptions {
 			// `react-dom/server`, and for DCR to alias that to `preact-render-to-string`.
 			// Then we can get rid of this line.
 			'preact-render-to-string': 'react-dom/server',
+			// Webpack 5 removed a lot of the nodejs polyfills including Buffer
+			// We rely on Buffer for our thrift client/server
+			Buffer: 'buffer',
```

Comment:
Will this also resolve `Buffer` to `buffer` in our server-side bundle? Should we maybe make this resolution apply for the client bundle only, since we probably want to use Node's `Buffer` server-side?

Reply:
Good point, I couldn't find places where we're using Buffer on the server but good to protect ourselves either way 👍